_posts/2019-07-11-austausch-aelterer-geraete.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title:  "Austausch/Aufwertung älterer Router"
+date:   2019-07-11 14:00:00
+categories: freifunkmuc
+---
+
+Wie ihr vielleicht schon im [Chat](https://chat.ffmuc.net) mitbekommen habt, nähern sich manche Geräte ihrem Lebensende. Diese Geräte fallen unter anderem  immer wieder damit auf, dass sie gelegentlich neustarten, eine generelle schlechte Performance zeigen und zu wenig Speicherplatz und Arbeitsspeicher besitzen.
+
+Gerade zu wenig Arbeisspecher stellt ein großes Problem für die Stabilität dar. Aber auch der Speicherplatz ist ein Problem, da OpenWRT (die Basis unserer Firmware) immer weiter wächst und diese Geräte nicht mehr genug Speicher für die Firmware besitzen.
+
+Aus diesem Grund wird es leider in naher Zukunft **keine** Updates - auch keine Sicherheitsupdates mehr für diese Geräte geben,  da schlicht die Firmware nicht mehr auf diese Geräte passt.
```

Comment:
Ich würde hinter Sicherheitsupdates noch ein "-" einfügen (beenden des Zusatzes).
Und eins der 2 Lehrzeichen hinter dem Komma entfernen.

"Kein Platz" ist redundant. "Aus diesem Grund" bezieht sich ja schon auf den Speicherplatz.
Der Teil nach dem Komma kann in dem Sinne auch einfach komplett weg.